app/models/concerns/authentication_concern.rb

```diff
@@ -16,9 +16,9 @@ def determine_cost
     end
 
     # Returns true if the given token matches the digest.
-    def authenticated?(remember_token, remember_digest = nil)
-      return false if remember_digest.nil?
-      BCrypt::Password.new(remember_digest).is_password?(remember_token)
+    def authenticated?(token, digest = nil)
+      return false if digest.nil?
+      BCrypt::Password.new(digest).is_password?(token)
```

Comment:
You can also use the `&&` operator which 'short-circuits' (returns left if true, else returns right)
```ruby
digest.present? && BCrypt::Password.new(digest).is_password?(token)
```